### DISCUSSION TITLE
Feature Request - Permanent Deep Sleep

### DISCUSSION THREAD

**Discussion by vsilvalopes** [darthcloud/BlueRetro#799] [Ideas]
Hi Blueretro community!

I mainly use BlueRetro installed internally in my systems.
The actual Deep Sleep option helps me when I want to use wired joysticks on my consoles.

But, I would like to know, if is possible to implement a "Super Deep Sleep" one that when you turn on and off the system, the ESP32 ports won't come on again, until you manually change the option to allow the ports to be used as expected by BlueRetro.

I'm thinking about that, because I only use the HW1 implementation of BlueRetro on my systems, installed internally and I don't have the knowledge and desire to go the HW2 route.

If this is something that is not possible, I'm probably going the hard switch route, with hardware switch button to interrupt the 5v line on the ESP32, so the module wont startup when I turn on the system.

Is this something possible?

Thanks in advance!

**Comment by darthcloud** [comment]
It's not impossible but it's not something I will personally work on. But as always I'm open to PR if someone else does it.

I think your best bet is to put a switch on the ESP supply.

**Comment by vsilvalopes** [comment]
No problem! If I knew how, I would gladly submit the PR. 
So I'm going the power switch route. 

Thanks for the reply!

**Comment by konwektor** [comment]
Can You just hold "BOOT: for short momment when powering up console, this puts BR in deep sleeep. no extra switches, no extra code....
ach code is already included in BR:
`/* If boot switch pressed at boot, trigger system on and goes to deep sleep */
if (sys_mgr_get_boot_btn() && !sys_mgr_get_power()) {
    sys_mgr_power_on();
    sys_mgr_deep_sleep();
}`
